lib/modules/pool/PoolDetail/PoolActivityChart/usePoolActivityChart.tsx

```diff
@@ -65,7 +65,11 @@ const getDefaultPoolActivityChartOptions = (
         formatter: (value: number) =>
           format(
             new Date(value * 1000),
-            `MMM d${differenceInDays(maxDate, minDate) < 1 ? ' HH:mm' : ''}`
+            `MMM d${
+              differenceInDays(secondsToMilliseconds(maxDate), secondsToMilliseconds(minDate)) < 1
```

Comment:
We also have a helper which might be a bit shorter?
```suggestion
              differenceInDays(secs(maxDate).toMs(), secs(minDate).toMs()) < 1
```